content/es/docs/languages/_index.md

```diff
@@ -8,9 +8,8 @@ default_lang_commit: 3815d1481fe753df10ea3dc26cbe64dba0230579
 ---
 
 La [instrumentación][] de código de OpenTelemetry es compatible con los
-lenguajes enumerados en la tabla de
-estados y lanzamientos presentada a continuación.
-También están disponibles implementaciones no oficiales para
+lenguajes enumerados en la tabla de estados y lanzamientos presentada a
+continuación. También están disponibles implementaciones no oficiales para
```

Comment:
Not sure why this is part of this PR, as @chalin said let's make sure that we do not have PRs with changes across localizations

Reply:
It seems that it was a lingering issue (of the Prettier not having been run over `es` pages).
Fixed by rebasing.

Reply:
Thanks for fixing it, @chalin ! It must have been introduced from the GH action I ran....